### ISSUE TITLE
Fix Angle Grinder installer

### ISSUE THREAD

**Issue by iainelder** [iainelder/dotfiles#9]
The installer started to fail on 2024-06-02.

[See log from CI run](https://github.com/iainelder/dotfiles/actions/runs/9335932429).

```
++ curl -Ss https://api.github.com/repos/rcoh/angle-grinder/releases/latest
++ command curl --netrc -Ss https://api.github.com/repos/rcoh/angle-grinder/releases/latest
++ curl --netrc -Ss https://api.github.com/repos/rcoh/angle-grinder/releases/latest
++ jq -r '.assets[] | select(.name == "agrind-x86_64-unknown-linux-musl.tar.gz") | .browser_download_url'
+ browser_download_url=
++ curl --silent --show-error --url '' --location --remote-name --write-out '%{filename_effective}'
++ command curl --netrc --silent --show-error --url '' --location --remote-name --write-out '%{filename_effective}'
++ curl --netrc --silent --show-error --url '' --location --remote-name --write-out '%{filename_effective}'
curl: (23) Failed writing received data to disk/application
+ download_filename=
```

**Comment by iainelder** [comment]
The release marked `latest` no longer contains the "musl" variant of the Linux asset.

I created the installer script on 2024-08-30.

The newest release then was [v0.19.2 from 2024-06-24](https://github.com/rcoh/angle-grinder/releases/tag/v0.15.0). It has three Linux assets:

* `agrind-x86_64-unknown-linux-gnu.tar.gz`
* `agrind-x86_64-unknown-linux-musl.tar.gz`
* `agrind-x86_64-unknown-netbsd.tar.gz`

Today the newest release is [v0.19.4 from 2024-06-01](https://github.com/rcoh/angle-grinder/releases/tag/v0.19.4). It has two Linux assets:

* `agrind-x86_64-unknown-linux-musl.tar.gz`
* ` agrind-x86_64-unknown-linux-gnu.tar.gz`.

The `latest` release is not the newest. It's [v0.19.3 also from 2024-06-24](https://github.com/rcoh/angle-grinder/releases/tag/v0.19.3). It has one Linux asset:

* `agrind-x86_64-unknown-linux-gnu.tar.gz`.

I don't know why I chose the "musl" variant over the "gnu" variant.

Maybe the easiest way to fix this is to use the "gnu" variant.

I may also report to the maintainer that the latest release is not the newest.
